Corona-Warn-App/src/main/res/xml/network_security_config.xml

```diff
@@ -1,2 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
-<network-security-config></network-security-config>
\ No newline at end of file
+<network-security-config xmlns:tools="http://schemas.android.com/tools">
+
+    <domain-config cleartextTrafficPermitted="false">
+        <!-- TODO Add Production Domain and add Backup Pin if applicable-->
+        <domain includeSubdomains="true">prodSubmit.com</domain>
+        <domain includeSubdomains="true">prodVerify.com</domain>
+        <pin-set expiration="2024-02-12" tools:ignore="MissingBackupPin">
+            <pin digest="SHA-256">c3jf+L8VIAFQnJJDM6Mfb4MtI1JnhVS8JwZHMwJj28M=</pin>
+        </pin-set>
+        <trust-anchors>
+            <certificates src="system" overridePins="false">
+            </certificates>
+        </trust-anchors>
+    </domain-config>
+
+    <domain-config cleartextTrafficPermitted="false">
+        <!-- TODO Add Production Domain and add Backup Pin if applicable-->
+        <domain includeSubdomains="true">prodDistribute.com</domain>
```

Comment:
`example.org`?

Reply:
Correct Pinning but still omitting URL (thats why we have the draft)